src/styles/blocks/full-width-image.scss

```diff
@@ -0,0 +1,7 @@
+.full-width-image {
+  &__content {
```

Comment:
The class name should be '.full-width-image__content' instead of '&__content' to maintain consistency and readability.